assets/js/components/Config/ModbusProxyModal.vue

```diff
@@ -1,27 +1,305 @@
 <template>
-	<YamlModal
+	<JsonModal
 		id="modbusProxyModal"
 		:title="$t('config.modbusproxy.title')"
 		:description="$t('config.modbusproxy.description')"
 		docs="/docs/reference/configuration/modbusproxy"
-		:defaultYaml="defaultYaml"
 		endpoint="/config/modbusproxy"
-		removeKey="modbusproxy"
-		size="md"
+		state-key="modbusproxy"
+		:store-values-in-array="true"
+		disable-remove
+		data-testid="modbusproxy-modal"
 		@changed="$emit('changed')"
-	/>
+	>
+		<template #default="{ values }: { values: ModbusProxy[] }">
+			<div class="mb-3">
+				<pre class="text-monospace">{{ ASCII_DIAGRAM }}</pre>
+				<h6>Network Connection</h6>
+				<p>
+					The device is directly connected via a native network connection (Modbus TCP).
+				</p>
+				<div
+					v-for="(connection, index) in values.filter(
+						(item) => 'URI' in item.Settings
+					) as ModbusProxy<ModbusProxyNetworkConnection>[]"
+					:key="index"
+				>
+					<div class="d-none d-lg-block">
+						<hr class="mt-5" />
+						<h5>
+							<div class="inner mb-3">Network Connection #{{ index + 1 }}</div>
+						</h5>
+					</div>
+
+					<FormRow
+						id="networkConnectionURI"
+						label="URI"
+						example="192.0.2.2:502"
+						help="The IP address and the port of the target device in common URI Scheme."
+					>
+						<input
+							id="networkConnectionURI"
+							v-model="connection.Settings.URI"
```

Comment:
```suggestion
							v-model="connection.settings.uri"
```
Lower case. See other published structures.

Reply:
Die anderen Felder (`Port`, `ReadOnly`) sind aber auch Upper Case:
Oder geht es darum, dass `Settings` wiederum ein Objekt enthält und deshalb klein zu schreiben ist?
https://github.com/evcc-io/evcc/blob/180145ed56dbfc11d52d071a030c4f2ef4fe382a/api/globalconfig/types.go#L59-L60